productImport/ontologyValidator/attributesValidator.go

```diff
@@ -68,6 +70,12 @@ func (v *Validator) validateAttributesAgainstRules(
 								isError = true
 							}
 
+							// units of measurement (UOM)
+							if isValid, errorMessage := v.isValidAttributeUoM(reportAttribute.UoM, attr); !isValid && errorMessage != "" {
```

Comment:
What happens if attribute is not valid and error message is empty?

Reply:
fixed